src/utils/outputSvg.ts

```diff
@@ -0,0 +1,67 @@
+import { select } from 'd3-selection';
+
+export type TOutputType = 'png' | 'blob';
+
+// Method to combine an svg specified by id with the InfoSum watermark and 
+// produce a blob or png output that can be used for download
+export const outputSvg = (
+  svgId: string,
+  width: number,
+  height: number,
+  callback: (outpuData: string | Blob | null) => void,
+  type: TOutputType = 'blob',
+) => {
+  // Select the first svg element
+  const svg: any = select(`svg#${svgId}`);
+  const serializer = new XMLSerializer();
+  // generate IMG in new tab
+  const svgStr = serializer.serializeToString(svg.node());
+  const watermark = require('../assets/Powered-By-InfoSum_DARK.svg') as string;
```

Comment:
it would be nice to parametrize this option

Reply:
Good call - I'll add in params for the sizing of this too